test/regress/cli/regress0/arith/issue5219-conflict-rewrite.smt2

```diff
@@ -2,7 +2,6 @@
 ; COMMAND-LINE: --theoryof-mode=term --nl-icp
 ; EXPECT: sat
 (set-logic QF_NRA)
-(set-option :produce-proofs true)
```

Comment:
I think we should keep `produce-proofs` set for benchmarks that are not "unsat", as this regression otherwise does not test proofs.  I believe benchmarks like these were added since a "sat" benchmark had an error e.g. in bookkeeping proofs.